packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -667,6 +667,28 @@ function processDebugValues(
   }
 }
 
+function handleRenderFunctionError(error: any): void {
+  // original error might be any type.
+  const isError = error instanceof Error;
+  if (isError && error.name === 'ReactDebugToolsUnsupportedFeatureError') {
+    throw error;
+  }
+  // If the error is not caused by an unsupported feature, it means
+  // that the error is caused by user's code in renderFunction.
+  // In this case, we should wrap the original error inside a custom error
+  // so that devtools can show a clear message for it.
+  const messgae: string =
```

Comment:
Curious why we want to use the original error message as the message for our custom error. Could we not just always have the error message for the wrapper error be `'Error rendering inspected component'`?

Reply:
I figured if we do have the original error message, it might be more helpful for users to see it. But semantics wise yes it's better to just use `'Error rendering inspected component'`.